client/packages/system/src/Patient/Components/PatientSearchInput/PatientSearchInput.tsx

```diff
@@ -32,7 +32,7 @@ export const PatientSearchInput: FC<NameSearchInputProps> = ({
       isOptionEqualToValue={(option, value) => option.name === value.name}
       width={`${width}px`}
       popperMinWidth={width}
-      value={value && { ...value, label: value.name }}
+      defaultValue={value && { ...value, label: value.name }}
```

Comment:
default value can't change after initial render - this was causing the original bug of patient name not updating when you had stale patient data.

By doing the wait until fetched after mount, we get the up to date patient data, then render the input 🎉